### PR TITLE
Hide previous and next buttons when there is only 1 iteration

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/v3-project-management/issues/133.

### CODE REVIEW

**Review by iHiD — APPROVED** [review]
Looks great, thanks :)

**Comment by iHiD** [comment]
Tests are failing.

File: app/javascript/components/mentoring/session/IterationsList.tsx [hunk 1]
```diff
@@ -52,6 +40,40 @@ export const IterationsList = ({
...
+          <IterationButton
+            key={iteration.idx}
+            iteration={iteration}
+            onClick={() => onClick(iteration)}
+            selected={current === iteration}
+          />
+        ))}
+      </nav>
+
+      {/* TODO: Move this into a component that can take either an icon or a character as the contents of --kb */}
```
Comment (ErikSchierboom) [inline app/javascript/components/mentoring/session/IterationsList.tsx:69]: 👍

**Review by ErikSchierboom — APPROVED** [review]

**Comment by iHiD** [comment]
Still failing :(

**Pull request merged**